Test failure with xarray 0.19.0

The tests with 0.19.0 fail with errors like:
```
>           integral = integral.integrate(dim=tcoord)
E           TypeError: DataArray.integrate() got an unexpected keyword argument 'dim'
```

Full output: https://download.copr.fedorainfracloud.org/results/davidsch/testing/fedora-rawhide-x86_64/02579853-python-xbout/builder-live.log.gz

--- a/xbout/boutdataset.py
+++ b/xbout/boutdataset.py
@@ -455,7 +455,7 @@ class BoutDatasetAccessor:
             if cumulative_t:
                 integral = integral.cumulative_integrate(coord=tcoord)
             else:
-                integral = integral.integrate(dim=tcoord)
+                integral = integral.integrate(coord=tcoord)
 
         return integral
 

